README.md

```diff
@@ -110,6 +110,38 @@ generated_text = model.generate("Once upon a time, ", n_predict=55)
 print(generated_text)
 ```
 
+## Interactive Mode
+
+If you want to run the program in interactive mode you can add the `grab_text_callback` function and set `interactive` to True in the generate function. `grab_text_callback` should always return a string unless you wish to signal EOF in which case you should return None.
+
+```py
+from pyllamacpp.model import Model
+
+def new_text_callback(text: str):
+    print(text, end="", flush=True)
+
+def grab_text_callback() -> str:
+    inpt = input()
+    # To signal EOF, return None
+    if inpt == "END":
+        return None
+    return inpt + "\n"
```

Comment:
Are you sure about `return inpt + "\n"` instead of `return inpt`? Because when I test it, only `return inpt` works correctly. It makes sense because the original `std::getline()` call also strips away `\n`.

Reply:
Thanks @nuance1979 
I think you are right, I will fix it.